entrypoint.sh

```diff
@@ -1,15 +1,18 @@
 #!/bin/sh
-set -e
 
 if [ -n "${GITHUB_WORKSPACE}" ] ; then
   cd "${GITHUB_WORKSPACE}/${INPUT_WORKDIR}" || exit
 fi
 
 export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
 
-misspell -locale="${INPUT_LOCALE}" . \
-  | reviewdog -efm="%f:%l:%c: %m" \
-      -name="linter-name (misspell)" \
+/usr/local/bin/phpcs.phar --config-set installed_paths /tmp/wpcs
+
+/usr/local/bin/phpcs.phar \
+  --report-checkstyle \
+  ${INPUT_PHPCS_ARGS:-\.} \
```

Comment:
📝 **[shellcheck]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Double quote to prevent globbing and word splitting. [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086)